code/datums/mind.dm

```diff
@@ -51,7 +51,6 @@
 	var/datum/changeling/changeling		//changeling holder
 	var/linglink
 	var/datum/vampire/vampire			//vampire holder
-	var/datum/abductor/abductor			//abductor holder
```

Comment:
`/datum/abductor/` is not a type, and this is not used for anything (other than to confuse people and get mysterious `error: mind.abductor: undefined type: mind.abductor` compilation errors).
Travis still passing is a proof of uselesness.